_locales/fr/messages.json

```diff
@@ -1044,12 +1050,18 @@
     "receiverRcInterpolationInterval": {
         "message": "Intervalle d'interpolation RC [ms]"
     },
+    "pidTuningDtermSetpointTransition": {
+        "message": "D Setpoint transition"
+    },
     "pidTuningPtermSetpoint": {
```

Comment:
This should be removed, it's not used any more, and was removed from the English file.